src/elements/icon-tile/src/stories.tsx

```diff
@@ -20,11 +20,20 @@ export const ExampleWithKnobs = () => {
     ['carInsurance', 'creditCards', 'loans', 'mortgages'],
     'carInsurance'
   )
+  const displayVariant = select(
+    'Display Variant',
+    ['horizontal', 'vertical'],
```

Comment:
import the display variant and use it here

Reply:
updated